packages/lesswrong/lib/rateLimits/constants.ts

```diff
@@ -27,13 +28,13 @@ const EA: {POSTS: PostAutoRateLimit[], COMMENTS: CommentAutoRateLimit[]} = {
   COMMENTS: [
     {
       ...timeframe('4 Comments per 30 minutes'),
-      karmaThreshold: 30,
+      isActive: user => user.karma <= 30,
       rateLimitType: "lowKarma",
       rateLimitMessage: "You'll be able to post more comments as your karma increases.",
       appliesToOwnPosts: true
     }, {
       ...timeframe('4 Comments per 30 minutes'),
-      downvoteRatioThreshold: .3,
+      isActive: (user, features) => features?.downvoteRatio >= 0.3,
```

Comment:
if `features` is optional, you can't perform a numeric comparison on a field you access via the optional chaining operator; you need to check that the left side of the comparison isn't undefined first.